openpype/hosts/maya/plugins/publish/extract_multiverse_usd.py

```diff
@@ -0,0 +1,205 @@
+import os
+
+import avalon.maya
+import openpype.api
+
+from maya import cmds
+
+
+class ExtractMultiverseUsd(openpype.api.Extractor):
+    """Extractor for USD by Multiverse."""
+
+    label = "Extract Multiverse USD"
+    hosts = ["maya"]
+    families = ["usd"]
+
+    @property
+    def options(self):
```

Comment:
This will create new dictionary everytime you'll access `self.options`. Why it's not just defined as class attribute so it's kept in memory?

Reply:
I think it is mimicked from another plug-in. E.g. [Maya FBX Extractor](https://github.com/pypeclub/OpenPype/blob/develop/openpype/hosts/maya/plugins/publish/extract_fbx.py#L38-L74).

Having it as a property does allow to potentially at some point return a different dictionary depending on the version of Multiverse (e.g. new features/options might only be added for specific Multiverse versions?)

Reply:
In that case it can return different attributes but not new dictionary.
```
class MyClass:
    _options_v1 = {...}
    _options_default = {...}
    
    @property
    def options(self):
        if get_version() == 1:
            return self._options_v1
        return self._options_default
```